uptane-standard.md

```diff
@@ -561,20 +561,22 @@ The Timestamp metadata SHALL contain the following information:
 
 Repository mapping metadata informs a primary ECU about which repositories to trust for images or image paths. Repository mapping metadata MUST be present on all primary ECUs, and MUST contain the following information:
 
-* A list of repository names and one or more URLs at which the named repository can be accessed. At a minimum, this MUST include the Director and Image repositories.
-* A list of mappings of image paths to repositories, each of which contains:
+* A list of repository names, each with one or more URLs at which the repository can be accessed.  The first repository name in this list SHALL be known as the Director repository.
+* A list of mappings of image paths to repositories.  Each mapping MUST contain:
     * A list of image paths. Image paths MAY be expressed using wildcards, or by enumerating a list, or a combination of the two.
-    * A list of repositories that MUST sign the targets metadata for the image paths.
+    * A list of repositories (at least two) that MAY sign the targets metadata for the image paths.  The first must always be the Director repository (the same repository that is listed first in the list of repository names).
+    * A Boolean "terminating" flag which indicates whether the client MUST NOT continue searching subsequent mappings (after failing to find the requested image from at least one of the repositories specified in this mapping, and given that this mapping matches the request image).
+    * A threshold which indicates the minimum number of repositories that MUST sign off the same non-custom targets metadata on any matching image.  This threshold MUST be at least two, and SHOULD be equal to the number of repositories listed in this mapping.
 
 For example, in the most basic Uptane case, the repository mapping metadata would contain:
 
-* The name and URL of the Director repository
-* The name and URL of the Image repository
-* A single mapping indicating that all images (`*`) MUST be signed by both the Director and Image repository
+* The name and URLs of the Director repository
+* The name and URLs of the Image repository
+* A single terminating mapping indicating that all images (`*`) MUST be signed by both the Director and Image repository (i.e., a threshold of 2 of 2 repositories)
 
 However, more complex repository mapping metadata can permit more complicated use cases. For example:
 
-* A second Director repository might be useful for fleet management of after-market vehicles, such as a rental car company that might wish to only install approved updates.
+* A third repository might be useful for fleet management of after-market vehicles, such as a rental car company that might wish to only install approved updates.
```

Comment:
I would leave actually calling additional repositories "directors" for the deployment considerations.  It creates the possibility of confusion here, where it has otherwise (I hope!) been very clear that there is one Director repository.  (See other comments)

Reply:
I think we have made it as clear as possible, we should be ok.